sys-cluster/ipvsadm/ipvsadm-1.31-r1.ebuild

```diff
@@ -14,11 +14,18 @@ SLOT="0"
 KEYWORDS="~amd64 ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86"
 IUSE="static-libs"
 
-RDEPEND=">=sys-libs/ncurses-5.2:*
+RDEPEND="
+	>=sys-libs/ncurses-5.2:*
```

Comment:
If linking against ncurses (I don't know if we are, check), this should be :=, not :*.